README.md

```diff
@@ -219,22 +219,25 @@ the `default_mappings` table.
 
 
 Much like [built-in pickers](#built-in-pickers), there are a number of
-[built-in actions](https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/actions.lua) you can pick from to remap your telescope buffer mappings:
-
+[built-in actions](https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/actions.lua) you can pick from to remap your telescope buffer mappings or create a new custom action:
```

Comment:
path changed it is now `lua/telescope/actions/init.lua`

Reply:
Nice, thanks